vpr/src/base/read_options.h

```diff
@@ -26,6 +26,12 @@ struct t_options {
     argparse::ArgValue<std::string> write_rr_graph_file;
     argparse::ArgValue<std::string> read_rr_graph_file;
 
+    argparse::ArgValue<std::string> write_placement_delay_lookup;
+    argparse::ArgValue<std::string> read_placement_delay_lookup;
+
+    argparse::ArgValue<std::string> write_lookahead;
+    argparse::ArgValue<std::string> read_lookahead;
```

Comment:
Should probably be `read/write_router_lookahead` for clarity.

Reply:
Fixed